content/en/docs/release-oversight/backend_queries.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Testing Backends For Availability"
```

Comment:
We should move this file under the `disruption-testing` folder as well

Reply:
> We should move this file under the `disruption-testing` folder as well

Yes, I'm cleaning things up now